### DISCUSSION TITLE
pika迁移数据到redis

### DISCUSSION THREAD

**Discussion by vb3328998** [pika/pika#1379] [General]
pika怎么把数据迁移到redis，用slaveof的方式不得行

**Comment by lukebakken** [comment]
This is Google's translation:

> How does pika migrate data to redis, it is not feasible to use slaveof

Pika is a library for communication with an AMQP 0.9.1 server like RabbitMQ. If you wish to transfer data to Redis from RabbitMQ, you will have to use a [Redis client library](https://github.com/redis/redis-py) and code the data migration yourself.
